db/seeds/muscle_group_descriptions.rb

```diff
@@ -0,0 +1,22 @@
+
+  ARMS =
+  "The muscles of the arms are one of the most sought after parts to develop. Your arm consists two thirds of triceps and one third of biceps, making the triceps muscle the larger of the two. Therefore, if you seek to gain more size in your arms, be sure to focus on the triceps to a greater degree. In men there is usually a strong emphasis placed on training the biceps over triceps. On the other hand, many women feel that they need to tone up th triceps area since over time, skin and fat begins to hang and look un-appealing with age. Toning up the triceps is the solution to this common problem. Complete arm development requires the training of both biceps and triceps."
+
+  SHOULDERS =
+  "The deltoid as a whole is composed of three small muscle heads, therefore is a relatively small muscle group. Basically, there are two types of exercises for the deltoids; presses and raises. These used in variations will train all three heads effectively. It is proven that low reps are best for all pressing movements and high reps are most effective for all raising movements."
+
+  BACK =
+  "The back is divided into three generalizations, the thickness, width and detail of the lower back. Of course, there are a variety of exercises available which target each area specifically. The two main movements or type of movements which stimulate the back muscles are the range of movement of the shoulders being pulled back or down. In addition, for the lower back, since it's function is that of a stabilizer, you must actually bend at the waist to stimulate its growth. The trapezius muscles which form the basis of the upper back work in opposition to the exercises of the back. However, the trapezius are trained as a secondary muscle for many exercises of other body parts. The exercises which are beneficial are shrugging movements."
+
+  LEGS =
+  "The thighs in general are tough mentally and physically to train, when done so correctly. It takes dedication and commitment to build balanced thighs. For the quadriceps as well as the hamstrings, it is very important to use a full range of motion in all exercises. While training quadriceps, it is important to incorporate basic foundation movements with isolating movements. While training the thighs you will also target your buttocks as a secondary muscle. This is especially effective for squatting and lunging exercises. In addition, hamstrings are a muscle which must be fully stretched at the extension phase of the movement while trained. High reps are most effective for isolation movements, where low rep are for compound exercises such as squats can be a good muscle builder. You must also take care to stretch and warmup prior to any exercise, especially the knees and other susceptible joints."
+
+  BUTTOCKS =
+  "Basically, most leg exercise will target the buttock directly or indirectly. Common movements such as squats and lunges have a profound effect on the strength and development of this region."
```

Comment:
Line is too long. [194/80]<br>Indent the first line of the right-hand-side of a multi-line assignment.<br>Freeze mutable objects assigned to constants.